.github/workflows/release.yml

```
@@ -0,0 +1,122 @@
+name: Release
+on:
+ pull_request:
+   branches:
+     - main
+   types:
+     - closed
+jobs:
+  tagging:
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.merged == true && startsWith(github.event.pull_request.head.ref, 'release/')
```

可能であれば複数行に分けたほうが見やすい

```suggestion
    if:
      github.event.pull_request.merged == true
      && startsWith(github.event.pull_request.head.ref, 'release/')
```

---

別コミットで修正

---

https://github.com/pfnet-research/pfhedge/pull/615/commits/f8287af4b04b5048ebe4b9afa9b998453053f3fb